web/app/build/build.component.scss

```diff
@@ -11,14 +11,16 @@ $TOOLBAR_HEIGHT:80px;
         .fci-build-logs {
             flex: 1;
             padding: 8px;
-            color: #859900;
+            color: #93a1a1;
             background-color: #073642;
+            font-family: 'Google Sans Monospace', sans-serif;
```

Comment:
Thanks! Let's make it 
```
'Google Sans Monospace', monospace;
```
for now. We likely want a default monospace font at all times.